setup.py

```diff
@@ -86,15 +86,34 @@ def is_platform_mac():
     if not _CYTHON_INSTALLED:
         raise ImportError('No supported version of Cython installed.')
     from Cython.Distutils import build_ext as _build_ext
+    from Cython import Tempita as tempita
```

Comment:
I'm not sure the vendored version of Tempita in Cython is public API. It might be better to add it as a build dependency.